test/object-store/util/test_file.cpp

```diff
@@ -171,11 +171,11 @@ SyncServer::SyncServer(const SyncServer::Config& config)
                    logger->set_level_threshold(realm::util::Logger::Level::TEST_ENABLE_SYNC_LOGGING_LEVEL);
                    m_logger.reset(logger);
 #else
-                   m_logger.reset(new TestLogger());
+                   m_logger.reset(new util::NullLogger());
```

Comment:
same here

Reply:
`TEST_ENABLE_SYNC_LOGGING` is not set, so logging is not requested. The `NullLogger` makes sense here.